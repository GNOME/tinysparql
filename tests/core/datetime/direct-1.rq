SELECT ("0001-01-01T00:00:00Z"^^xsd:dateTime AS ?d1) ("2020-01-01T00:00:00Z"^^xsd:dateTime AS ?d2) ("9999-01-01T00:00:00Z"^^xsd:dateTime AS ?d3) {}