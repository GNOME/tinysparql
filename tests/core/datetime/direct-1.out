"0001-01-01T00:00:00Z"	"2020-01-01T00:00:00Z"	"9999-01-01T00:00:00Z"
